src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -16,18 +18,19 @@
 
 namespace {
 auto is_supported_tensor(const ngraph::descriptor::Tensor& t) -> bool {
-    // TODO: Add support of all supported by common tokenization element types
-    //       return ngraph::snippets::pass::TokenizeSnippets::supported_element_types.count(input.get_element_type()) != 0;
-    //       Also only 4D is supported at the moment
-    return t.get_element_type() == ngraph::element::f32 && t.get_partial_shape().is_static() && t.get_shape().size() == 4;
+    // TODO: Add support of non-4D tensors
```

Comment:
I would rather remove todos like this, because no one will be looking through the sources to find `todos` and prioritize (or create) tasks based on them.
It's probably better to provide an explanatory remark instead, like:
`Only 4D tensors are currently supported. Will be fixed in [Ticket id]`
Otherwise code with a lot of todo's may look not reliable.

The same is relevant for L25 and on L131